docs/releasing/testing-and-linting.md

```diff
@@ -46,9 +52,32 @@ To lint the whole codebase, run:
 npm run lint
 ```
 
-This will trigger [ESLint](https://eslint.org) (using [JavaScript Standard Style](https://standardjs.com)) and [Stylelint](https://stylelint.io) for linting.
+This will run the following checks:
+
+1. [EditorConfig](https://editorconfig.org)
+2. [Prettier](https://prettier.io)
+3. [ESLint](https://eslint.org) (using [JavaScript Standard Style](https://standardjs.com))
+4. [Stylelint](https://stylelint.io) (using [GDS Stylelint Config](https://github.com/alphagov/stylelint-config-gds))
+
+Where possible, issues will be fixed automatically on `git commit` using [Husky](https://typicode.github.io/husky/) and [`.lintstagedrc.js`](/.lintstagedrc.js).
+
+To commit changes without automatic checks use `git commit --no-verify`.
+
+### Running only EditorConfig linting
+
+```shell
+npm run lint:editorconfig
+```
+
+See [.editorconfig](/.editorconfig) for details.
+
+### Running only Prettier linting
+
+```shell
+npm run lint:prettier
+```
```

Comment:
**issue** I think it's worth mentioning the `--write` option for Prettier similarly to how we mention the `--fix` options for Stylelint and Eslint. I'm not sure if it's compatible with `--check`, though, so we may have to move that flag in the `lint:prettier` script rather than `lint:prettier:cli` 😊